### PR TITLE
make clean target to clean opt folder

### PR DESCRIPTION
**Pull request by Sunnatillo**

### CODE REVIEW

File: host_cleanup.sh [hunk 1]
```diff
@@ -64,3 +64,7 @@ fi
 sudo rm -rf /var/log/libvirt/qemu/*serial0.log*
 
 rm -rf "${HOME}"/.cluster-api
```
Comment (tuminoid) [inline host_cleanup.sh:66]: Maybe add removing CAPI 1.5 config folder from the new location as well

**Comment by Sunnatillo** [comment]
/test-ubuntu-e2e-feature-main

test

**Comment by tuminoid** [comment]
@Sunnatillo we should get this merged, or was there some unexpected downsides?

**Pull request closed without merging**